docs/Configuring.md

```diff
@@ -506,27 +507,52 @@ When `invalidate-bad-cache-results=true` (default: `true`), Cromwell will invali
 
 Cromwell also accepts [Workflow Options](wf_options/Overview#call-caching-options) to override the cache read/write behavior.  
 
+#### Call cache strategy options
+
+* hash based options. These read the entire file. These strategies work with containers.
+    * `xxh64`. This uses the 64-bit implementation of the [xxHash](https://www.xxhash.com)
+             algorithm. This algorithm is optimized for file integrity hashing and provides a more than 10x speed improvement over
+             md5. 
+    * `md5`. The well-known md5sum algorithm
+* Path based options. These are based on filepath. Extremely lightweight, but only work with the `soft-link` file 
+caching strategy and can therefore never work with containers.
+    * `path` creates a md5 hash of the path.
+    * `path+modtime` creates a md5 hash of the path and its modification time.
+* Fingerprinting. This strategy works with containers.
+    * `fingerprint` tries to create a fingerprint for each file by taking its last modified time (milliseconds since
+       epoch in hexadecimal) + size (bytes in hexadecimal) + the xxh64 sum of the first 10 MB of the file. It is much
```

Comment:
Was '10'MB based on anything specific or is it just a reasonable sounding number?

eg is there a good answer to "why not 5mb" or "why not 20mb"?

Reply:
Well 5MB and 20 MB are the same order of magnitude. So at that point it is arbitrary. However, "why not 1 MB?" or "why not 100 MB?" can be answered.  I have added a footnote explaining the choice.